laboratory-service/src/main/java/com/raf/si/laboratoryservice/controller/WorkOrderController.java

```diff
@@ -0,0 +1,70 @@
+package com.raf.si.laboratoryservice.controller;
+
+import com.raf.si.laboratoryservice.dto.request.order.*;
+import com.raf.si.laboratoryservice.dto.response.order.*;
+import com.raf.si.laboratoryservice.services.WorkOrderService;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+
+@Slf4j
+@RestController
+@CrossOrigin
+@RequestMapping("/order")
+public class WorkOrderController {
+    private final WorkOrderService orderService;
+
+    public WorkOrderController(WorkOrderService orderService) {
+        this.orderService = orderService;
+    }
+
+    @PreAuthorize("hasRole('ROLE_LAB_TEHNICAR') or hasRole('ROLE_VISI_LAB_TEHNICAR')")
+    @PostMapping("/create")
+    public ResponseEntity<CreateOrderResponse> createWorkOrder(@Valid @RequestBody CreateOrderRequest orderRequest){
+        CreateOrderResponse response = orderService.createOrder(orderRequest);
+        System.out.println(response.toString());
+        return ResponseEntity.ok(response);
+    }
+
+    @PreAuthorize("hasRole('ROLE_NACELNIK_ODELJENJA') or hasRole('ROLE_DOKTOR_SPEC') or hasRole('DOKTOR_SPEC_POV')")
+    @GetMapping("/history")
```

Comment:
Get requestovi obicno nemaju body, ako ces da koristis body napravi da budu post. Takodje, nemoj id, lbp i takve stvari da saljes u requestu (mozes ako imas vise stvari u requestu, ali sigurno ne ako je samo to polje requesta). Bolje je da napravis kao path varijablu, mozes da vidis kako u patientController klasi.